projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html

```diff
@@ -59,14 +59,14 @@ <h2 class="heading-h2 error-summary-heading" id="edit-case-event_error-summary-h
                       <th id="case-viewer-field-label" *ngIf="!isFieldToHaveNoLabel(field)">
                         <div class="case-viewer-label text-16">{{field.label}}</div>
                       </th>
-                      <td>
+                      <th>
```

Comment:
@johnbenjamin-hmcts - why the change from `<td>` to `<th>`? If I remember correctly, the same change made last time caused problems elsewhere and I had to revert it

Reply:
@Daniel-Lam if `<td>` then the case file view tab content is rendered only a quarter of the screen. Had to change it to `<th>` to fix this issue.

Reply:
@johnbenjamin-hmcts - we will need to test with other tabs because I'm sure this caused a problem in the past